modules/simple-payments/paypal-express-checkout.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This PaypalExpressCheckout global is included by wp_enqueue_script( 'paypal-express-checkout' );
+ * It handles communication with Paypal Express checkout and public-api.wordpress.com for the purposes
+ * of simple-payments module.
+ */
+
+/* global paypal */
+/* exported PaypalExpressCheckout */
+/* jshint unused:false */
+var PaypalExpressCheckout = {
+	getCreatePaymentEndpoint: function( blogId ) {
+		return 'https://public-api.wordpress.com/wpcom/v2/sites/' + blogId + '/simple-payments/paypal/payment';
+	},
+	getExecutePaymentEndpoint: function( blogId, paymentId ) {
+		return 'https://public-api.wordpress.com/wpcom/v2/sites/' + blogId + '/simple-payments/paypal/' + paymentId + '/execute';
+	},
+	getNumberOfItems( field, enableMultiple ) {
+		var numberField, number;
+		if ( enableMultiple !== '1' ) {
+			return 1;
+		}
+		numberField = document.getElementById( field );
+
+		if ( ! numberField ) {
+			return 1;
+		}
+		number = Number( numberField.value );
+
+		if ( isNaN( number ) ) {
+			return 1;
+		}
+		return number;
+	},
+	renderButton: function( blogId, buttonId, domId, enableMultiple ) {
+		if ( ! paypal ) {
+			throw new Error( 'PayPal module is required by PaypalExpressCheckout' );
```

Comment:
Not sure we want to throw a JS error here. Wouldn't it stop other JS execution on a page? Maybe a bit destructive in that case.